scripts/cppcheck.sh

```diff
@@ -42,7 +42,7 @@ ret_code=0
 cat ${LOG_FILE} | grep -v -e "syntaxError," -e "cppcheckError," > ${LOG_FILE}.tmp
 mv ${LOG_FILE}.tmp ${LOG_FILE}
 
-for category in "style" "performance" "portability"; do
+for category in "style" "performance" "portability" "warning"; do
```

Comment:
perhaps create an issue to remember warnings are excluded for now ?

Reply:
Ok, done in https://github.com/qgis/libdxfrw/issues/31